Cargo.toml

```diff
@@ -6,3 +6,9 @@ members = [
     "mev-build-rs",
     "mev-rs",
 ]
+
+[patch."https://github.com/ralexstokes/ethereum-consensus"]
+ethereum-consensus = { git = "https://github.com/ralexstokes//ethereum-consensus", rev = "a8110af76d97bf2bf27fb987a671808fcbdf1834" }
+
+[patch."https://github.com/ralexstokes/ssz-rs"]
+ssz-rs = { git = "https://github.com/ralexstokes//ssz-rs", rev = "cb08f18ca919cc1b685b861d0fa9e2daabe89737" }
```

Comment:
I think we do also want to pin `beacon-api-client` as this was the issue that launched this whole arc